.gitignore

```diff
@@ -23,6 +23,8 @@ kube_burner*
 .pydevproject
 .settings
 .idea
+.vscode
+config/debug.yaml
```

Comment:
@tsebastiani  think config/debug.yaml and others below in (env and chaos dir) might not be candidates for .gitignore, since it is based on user behavior, if I am not mistaken.